### DISCUSSION TITLE
Team up

### DISCUSSION THREAD

**Discussion by quantrpeter** [antlr/grammars-v4#4241] [General]
Hi All
We are Hong Kong Programming Society and now want to create a php-like scripting language for web dev running on top of JVM. Is anybody want to team up? We are serious doing this project, now in a super early stage and want to form a team. We need people familar with antlr, especially converting lang-definition into antlr grammer, defining the language structure and all other stuff. We also need JVM bytebode guys too. We are not zero technical but we feel we are not enough for everything. We will hold a meeting roughly every 2 weeks to sync up everybody.
If you are interest, plesae leave a message

thanks
Peter

**Comment by kaby76** [comment]
Watch out for grammar ambiguities, which is a sign that the grammar needs work. I have a bunch of tools coming along in Trash to test and display ambiguity, e.g. `trperf file-to-parse -c Fdra | sort -k4 -r -n | head` (parse the input file, output `File-name/decision/rule-name/#ambiguities`, and sort by `#ambiguities`.

**Reply by quantrpeter** [reply]
thanks Buddy
